File: training/php/src/Domain/Model/Speed.php
<?php

declare(strict_types=1);

namespace De\Sots\CellarsAndCentaurs\Domain\Model;

final class Speed
{
    public function __construct(private int $feetPerRound)
    {
    }

    public function getFeetPerRound(): int
    {
        return $this->feetPerRound;
    }

    public function setFeetPerRound(int $feetPerRound): void
    {
        $this->feetPerRound = $feetPerRound;
    }
}

File: sample-projects/java/src/main/java/de/sots/cellarsandcentaurs/domain/model/Speed.java
package de.sots.cellarsandcentaurs.domain.model;

public class Speed {
    private int feetPerRound;

    public Speed(int feetPerRound) {
        this.feetPerRound = feetPerRound;
    }

    public int getFeetPerRound() {
        return feetPerRound;
    }

    public void setFeetPerRound(int feetPerRound) {
        this.feetPerRound = feetPerRound;
    }
}
